packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/xcpd-request.ts

```diff
@@ -28,10 +29,18 @@ export function transformIti55RequestToPatientResource(
     country: addr.country ? String(addr.country) : undefined,
   }));
 
-  const telecom = toArray(queryParams.patientTelecom?.value).map(tel => ({
-    system: "phone",
-    value: tel._value,
-  }));
+  // TODO calling email and phone both phone
+  const telecom = toArray(queryParams.patientTelecom?.value)
```

Comment:
we were setting both email and phone to phone, and then nuking the email during phone validation